website/docs/api/large-language-models.mdx

```diff
@@ -302,6 +303,171 @@ max_n_words = 20
+>
+> ```ini
+> [initialize.components.llm.candidate_selector.kb_loader]
+> @llm_misc = "spacy.KBObjectLoader.v1"
+> # Path to knowledge base directory in serialized spaCy pipeline.
+> path = ${paths.el_kb}
+> # Path to spaCy pipeline. If this is not specified, spacy-llm tries to determine this automatically (but may fail).
+> nlp_path = ${paths.el_nlp}
+> # Path to file with descriptions for entity.
+> desc_path = ${paths.el_desc}
+> ```
+
+| Argument          | Description                                                                                                                                                                                                                         |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `path`            | Path to KB file. ~~Union[str, Path]~~                                                                                                                                                                                               |
+| `nlp_path`        | Path to serialized NLP pipeline. If None, path will be guessed. ~~Optional[Union[Path, str]]~~                                                                                                                                      |
+| `desc_path`       | Path to file with descriptions for entities. ~~int~~                                                                                                                                                                                |
+| `ent_desc_reader` | Entity description reader. Defaults to an internal method expecting a CSV file without header row, with ";" as delimiters, and with two columns - one for the entitys' IDs, one for their descriptions. ~~Optional[EntDescReader]~~ |
+
+##### spacy.KBFileLoader.v1 {id="kb-file-loader-v1"}
+
+Adheres to the `InMemoryLookupKBLoader` interface required by
+[`spacy.CandidateSelector.v1`](#candidate-selector-v1). Loads a knowledge base
+from a knowledge base file. The KB .yaml file has to stick to the following
+format:
+
+```yaml
+entities:
+  # The key should be whatever ID identifies this entity in your knowledge base.
+  ID1:
+      name: "..."
+      desc: "..."
+  ID2:
+      ...
+# Data on aliases in your knowledge base - e. g. "Apple" for the entity "Apple Inc.".
+aliases:
+  - alias: "..."
+    # List of all entities that this alias refers to.
+    entities: ["ID1", "ID2", ...]
+    # Optional: prior probabilities that this alias refers to the n-th entity in the "entities" attribute.
+    probabilities: [0.5, 0.2, ...]
+  - alias: "..."
+    entities: [...]
+    probabilities: [...]
+  ...
+```
+
+See
+[here](https://github.com/explosion/spacy-llm/blob/main/spacy_llm/tests/tasks/misc/el_kb_data.yml)
```

Comment:
This link is broken and should refer to https://github.com/explosion/spacy-llm/blob/develop/usage_examples/el_openai/el_kb_data.yml instead, or rather the version of it on `main` once `develop` is merged in.

Reply:
The link will work once `develop` has been merged into `main` again. Since this PR will be merged into `docs/llm_develop`, I think that should be fine?

Reply:
I think it's still better to refer to the file in the `usage_examples` than the one in the test suite, though.